config/config.example.yml

```diff
@@ -203,12 +203,13 @@ https_only: false
 # -----------------------------
 
 ##
-## Enable/Disable the "Popular" tab on the main page.
+## Enable/Disable specific pages on the main page.
+## Example:
+## pages_enabled:
+##   trending: true
+##   popular: true
+##   search: true
```

Comment:
To remain consistent with the rest of the file (a single `#` is used to show the default value of a config option that is not mandatory):
```suggestion
## Enable/Disable specific pages on the main page.
##
#pages_enabled:
#  trending: true
#  popular: true
#  search: true
```